tidb_binlog/binlogctl/nodes.go

```diff
@@ -63,22 +54,22 @@ func unregisterNode(urls, kind, nodeID string) error {
 		return errors.Trace(err)
 	}
 
-	nodes, err := registry.Nodes(context.Background(), nodePrefix[kind])
+	nodes, err := registry.Nodes(context.Background(), node.NodePrefix[kind])
 	if err != nil {
 		return errors.Trace(err)
 	}
 
 	for _, n := range nodes {
 		if n.NodeID == nodeID {
-			if n.IsAlive {
+			if n.State != node.Unknow {
```

Comment:
Unknow or Alive? we don't need unregister, maybe we need `pause` and `offline`

Reply:
if we want pause or offline pump, I think we should send http request to pump, rather than change  the status in etcd.

Reply:
yep, we need `binlogctl` to offline and puase pump, in addition we also need a work around solution to handle various abnormal conditions, like change etcd information

Reply:
now pump don't support state transform now, I will do it later.